contributingGuides/API.md

```diff
@@ -43,14 +43,16 @@ The data will automatically be sent to the user via Pusher.
 #### WRITE Response Errors
 When there is an error on a WRITE response (`jsonCode!==200`), the error must come back to the client on the HTTPS response. The error is only relevant to the client that made the request and it wouldn't make sense to send it out to all connected clients.
 
-Error messages should be returned and stored as a String under the `error` property. If absolutely needed, additional error properties can be stored under other, more specific fields that sit at the same level as `error`:
+Error messages should be returned and stored as an object under the `errors` property, keyed by a [microtime](https://www.php.net/manual/en/function.microtime.php). If absolutely needed, additional error properties can be stored under other, more specific fields that sit at the same level as `errors`:
```

Comment:
> keyed by a microtime

I think you should mention that it is an int, and update it to use `Onyx::getErrorMicroTime()` instead. We can wait until that PR is merged.